Tools/LibEnsemble/warpx_simf.py

```diff
@@ -85,12 +85,12 @@ def run_warpx(H, persis_info, sim_specs, libE_info):
     # Safety
     time.sleep(0.2)
 
-    try:
-        # Get output from a run and delete output files
-        warpx_out = read_sim_output(task.workdir)
-    except Exception:
-        warpx_out = np.nan
-        print('Warning - output is Nan')
+    # Get output from a run and delete output files
+    warpx_out = read_sim_output(task.workdir)
+
+    # Excluding results - NAN - from runs where beam was lost
+    if (warpx_out[0] != warpx_out[0]):
+        print(task.workdir, ' output led to NAN values')
```

Comment:
```suggestion
    # Excluding results - NaN - from runs where beam was lost
    if (warpx_out[0] != warpx_out[0]):
        print(task.workdir, ' output led to NaN values (beam loss)')
```

Reply:
(totally optional, can be a follow-up if it's useful or be ignored)

Reply:
Sorry @ax3l , I only saw this now. Yes, it's a good idea.
I'll include it in a next PR!

Reply:
Sounds good - the looked so minor to me that I though you can choose and I'll merge this useful update already :)

Reply:
There are prints next to the ifs that test either "fails" beam loss or no plotfiles.
I am preparing a branch for this then.
What do you think of:
`print(task.workdir, ' output led to NaN values (beam was lost or run did not finish)')`

Reply:
looks great, feel free to break the string into multiple lines as well.